packages/neo-one-client/src/__tests__/provider/neoone/JSONRPCClient.test.js

```diff
@@ -0,0 +1,161 @@
+/* @flow */
+import BigNumber from 'bignumber.js';
+
+import { keys, transactions } from '@neo-one/client-core/src/__data__';
+
+import JSONRPCClient from '../../../provider/neoone/JSONRPCClient';
+import { RelayTransactionError } from '../../../errors';
+
+jest.mock('isomorphic-fetch');
```

Comment:
I think this can be removed, we're just forwarding calls to the underlying provider and no `fetch` is happening.